static/js/page/dokka-template/FilterButton/index.jsx

```diff
@@ -0,0 +1,13 @@
+import cn from 'classnames';
+import Button from "@rescui/button";
+import useMediaMatch from "use-match-media";
+
+import "./filter-button.scss";
+
+function FilterButton({ className, children: name, ...props }) {
+    const isSmall = useMediaMatch("(max-width: 840px)");
```

Comment:
It is doesn’t work for me. 
I got  ` window is not defined` error running the script.
I guess useMediaMatch call should we wrapped by condition 
`if (typeof window !== `undefined`) {`.

Also, notice that this error didn't break [the build](https://buildserver.labs.intellij.net/buildConfiguration/Kotlin_KotlinSites_KotlinlangTeamcityDsl_PrepareCustomDokkaTemplates/229755682?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildDeploymentsSection=true&showLog=229755682_1695_1631&logView=flowAware), we need to figure out why it is successful despite the error.